hearth/lib/hearth/validator.rb

```diff
@@ -41,5 +41,20 @@ def self.validate_unknown!(struct, params, context:)
       raise ArgumentError,
             "Unexpected members: [#{unknown.join(', ')}]"
     end
+
+    # draft of what validate_range *could* look like
+    # valid_range param consists of min and max keys
+    def self.validate_range(value, valid_range, context:)
```

Comment:
What about using kwargs for min and max?
```
self.validate_range(value, min:, max:, context:)
```
Makes it easy to use like: 
```
validate_range(value, min: 0, max: 10000)
```

Reply:
Ahh, yes. That would make sure we have `:min` and `:max` passed in.